frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.module.css

```diff
@@ -10,7 +10,6 @@
   padding: 0.25em 0.65em;
   border-radius: 6px;
   min-width: 35px;
-  color: var(--mb-color-brand);
```

Comment:
Is this removed because `--mb-color-brand` doesn't change in SDK context? That might explain the console error I saw.

To give some context. We update [colors in `pallette.ts`](https://github.com/metabase/metabase/blob/1d1bcdff9f0ff33d3bb61a3069017d7e71f9da02/enterprise/frontend/src/metabase-enterprise/whitelabel/lib/whitelabel.js#L4-L10), when we first load the app, so imagine users configure Metabase appearance settings by changing colors there.

I see we're using `color("brand")` directly instead of referring to Mantine theme color. https://github.com/metabase/metabase/blob/1d1bcdff9f0ff33d3bb61a3069017d7e71f9da02/enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx#L39

I think we should fix that, but if this doesn't have anythign to do with `--mb-color-brand` doesn't respect the SDK theme then you can ignore this comment for now.

Reply:
Nice, I wasn't aware that we were providing CSS variables in the SDK component wrapper. Let me fix the wrapper. Thanks!

Reply:
@WiNloSt Another reason here is that as we are moving towards Mantine, I think we can use Mantine's `c` prop instead of setting it here. Ideally it would be good to move every property to Mantine, but I think we can only migrate the one we need for now.